readme.md

```diff
@@ -49,28 +50,30 @@ Select 1:
 [] Selects all li's which are directly inside a ul of class dropdown (children)
 [] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
```

Comment:
[] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)